### PR TITLE
bug fixes and compatibility issues

### PR DESCRIPTION
**Pull request by pavanramkumar**

### CODE REVIEW

File: pyglmnet/pyglmnet.py [hunk 1]
```diff
@@ -607,7 +607,8 @@ def fit(self, X, y):
 
             # Update active set
             if self.solver == 'cdfast':
-                ActiveSet[beta[1:] == 0] = 0
+                ActiveSet[beta == 0] = 0
+                ActiveSet[0] = 1.
```
Comment (jasmainak) [inline pyglmnet/pyglmnet.py]: what is the shape of `ActiveSet`? n_features + 1?

Comment (pavanramkumar) [reply pyglmnet/pyglmnet.py]: yes. though we never deactivate the intercept because it's not regularized. can be reimplemented with size `n_features`

Comment (jasmainak) [reply pyglmnet/pyglmnet.py]: oh yes, as you like. Maybe a comment would do, or reimplementation with `n_features` is also fine. You need to make Travis happy too

**Comment by jasmainak** [comment]
@pavanramkumar did you get a chance to try the PR locally?

**Pull request merged**